test/unit/index.js

```diff
@@ -125,11 +125,52 @@ test('getCompatibility - with flags, testModeCompatibilityLevel and override ver
   t.true(n.isDone());
 });
 
-test('getCompatibility - with testMode true', async function(t) {
+test('getCompatibility - mode invalid input errors', async function(t) {
+  const vin = 'fake_vin';
+  const scope = ['read_location', 'read_odometer'];
+
+
+  const err = await t.throwsAsync(smartcar.getCompatibility(vin, scope, 'US', {
+    clientId: 'clientId',
+    clientSecret: 'clientSecret',
+    version: '6.6',
+    mode: 'pizzapasta',
+  }));
+  const message = 'mode MUST be one of the following '
+    + '\'test\', \'live\', \'simulated\'';
+  t.is(err.message, message);
```

Comment:
```suggestion
  t.is(
    err.message,
    "mode MUST be one of the following 'test', 'live', 'simulated'",
  );
```
Slightly easier to read vs string concat and escaping